unoR3/unoR3.ino

```diff
@@ -1,39 +1,47 @@
+#include <HomeAutomationLight.h>
+#include <HomeAutomationTimer.h>
 #include <dht.h>
 
 dht DHT;
 
-#define DHT11_PIN 5
-#define HEATER 7
-#define WINDOW_SENSOR_PIN 8
-#define MAINLIGHT 11
-#define MIRRORLIGHT 10
-#define LAMPLEFT 2
-#define LAMPRIGHT 3
-#define DOOR_SENSOR_PIN 9
+const int dht_pin = 5;
+const int heater_pin = 7;
+const int window_sensor_pin = 8;
+const int main_light_pin = 11;
+const int mirror_lamp_pin = 10;
+const int left_lamp_pin = 2;
+const int right_lamp_pin = 3;
+const int door_sensor_pin = 9;
```

Comment:
I made all these const ints as its 'safer', with `#define` you could write `DOOR_SENSOR PIN foobar` and you would either get confusing error or some odd behaviour. 

now if you try and set one of these to anything other than a number you will get a sensible error message when you compile. `const` just means that the value of this variable can never be changed